libs/platform/src/lib/components/table/table.component.ts

```diff
@@ -1771,4 +1744,17 @@ export class TableComponent<T = any> extends Table implements AfterViewInit, OnD
                 .subscribe(() => this.recalculateTableColumnWidth())
         )
     }
+
+    /** @hidden */
+    private _isRowNavigatable(row: T, rowNavigationDisabled: string | boolean): boolean {
+        if (!row) {
+            return false;
+        }
+
+        if (isString(rowNavigationDisabled)) {
+            return !!row[rowNavigationDisabled];
+        }
+
+        return !!rowNavigationDisabled;
```

Comment:
From this code if `rowNavigationDisabled` means `isRowNavigatable`.
Looks illogical to me.

Reply:
Great catch, agree!

Reply:
Now if found that this is different things.

We may have not set "rowNavigatable" and then call "setRowNavigation(rowIndex, rowNavigationState)" with boolean value - in that case `return !!this.rowNavigatable` won't calculate right value for the `row.navigatable`.

Reply:
Naming, improved, thank you!

Reply:
let's change rowNavigationDisabled to rowNavigatable